txauthor.go

```diff
@@ -35,12 +35,17 @@ func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
 	tx.sendFromAccount = uint32(accountNumber)
 }
 
```

Comment:
Add comment

Reply:
The comments on the exported functions is sorted out here in this Pr
https://github.com/raedahgroup/dcrlibwallet/pull/116.